community/pmc-tool/main/collector/queries.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from main.collector.db import (
+    fetch_first_pr_authors_last_week,
+    fetch_first_pr_date,
+    fetch_pull_requests,
+    fetch_issues,
+)
+from main.collector.validations import assert_equal
+
+
+def verify_pull_requests(conn, pr_authors_count):
+    pr_results = fetch_pull_requests(conn)
+    assert_equal(pr_results, pr_authors_count)
+    return pr_results
+
+
+def verify_issues(conn, issue_authors_count):
+    issue_results = fetch_issues(conn)
+    assert_equal(issue_results, issue_authors_count)
+    return issue_results
+
+
+def get_first_pr_authors_last_week(conn):
+    return fetch_first_pr_authors_last_week(conn)
```

Comment:
this is not called anywhere, right? can we remove it?

Reply:
Retrieves the first pull request authors in the last week. called in views.py/ render_results